x-pack/plugins/security_solution/server/lib/timeline/routes/timelines/persist_favorite/index.ts

```diff
@@ -60,14 +60,8 @@ export const persistFavoriteRoute = (router: SecuritySolutionPluginRouter) => {
             timelineType || TimelineTypeEnum.default
           );
 
-          const body: PersistFavoriteRouteResponse = {
-            data: {
-              persistFavorite: timeline,
-            },
-          };
-
           return response.ok({
-            body,
+            body: timeline,
```

Comment:
This may not work consistently as the result of `persistTimeline` isn't always successful. While it may return a timeline, it's just a default timeline and the persist may not actually have been successful: https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/server/lib/timeline/saved_object/timelines/index.ts#L407-L425

Reply:
Nice catch, fixed here: https://github.com/elastic/kibana/pull/200633/commits/267c0859548b608deb810dbe09d00eede9ce10f3